src/web3/MassaAmount.ts

```diff
@@ -0,0 +1,78 @@
+import BigNumber from 'bignumber.js';
+
+export enum MASSA_UNIT {
```

Comment:
[Nit-picking]As type script is a typed language, do we need to write anum value in capital letter? I have the feeling that the code is yelling at me :-)